rules-emerging-threats/2019/Malware/Ursnif/registry_add_malware_ursnif.yml

```diff
@@ -25,6 +25,8 @@ detection:
             - '\SOFTWARE\AppDataLow\Software\Microsoft\RepService\'
             - '\SOFTWARE\AppDataLow\Software\Microsoft\IME\'
             - '\SOFTWARE\AppDataLow\Software\Microsoft\Edge\'
+            - '\SOFTWARE\AppDataLow\Software\Microsoft\AppDataLow'
+            - '\Software\AppDataLow\Software\Microsoft\Software'
     condition: selection and not filter
```

Comment:
It seems we have so many filters for this rule to be of major indication. The trend micro blog showcase the use of a GUID like name convention. Better change the rule to reflect that than filtering this many things.